src/Approximators/RangeApproximators/helpers/power_its.jl

```diff
@@ -1,15 +1,30 @@
+"""
+    RLinearAlgebra.econQ!(A::AbstractMatrix)
+
+Function that returns the skinny Q matrix from a QR factorization.
+
+## Arguments
+ - `A::AbstractMatrix`, a matrix being decomposed.
+
+## Returns 
+ - Edits `A` in place with householder reflectors and returns the skinny QR.
+"""
+function econQ!(A::AbstractMatrix)
+    return Array(qr!(A).Q)
+end
```

Comment:
@nathanielpritchard great! I wonder if you might need the econQR outside Approximators. In such case consider relocating it (not urgent, we can do it if this case materializes).